packages/core/core/test/fixtures/common-targets/package.json

```diff
@@ -3,7 +3,7 @@
   "module": "dist/module/index.js",
   "browser": "dist/browser/index.js",
   "engines": {
-    "node": ">= 8.0.0"
+    "node": ">= 10.0.0"
```

Comment:
Should these be kept at >= 8? Was there a particular reason for including engines here?